docs/src/modules/java/pages/quickstart/cr-value-entity-kafka-java.adoc

```diff
@@ -126,11 +126,14 @@ To build and publish the container image and then deploy the service, follow the
 kalix auth login
 ----
 
-. Use the `deploy` target to build the container image, publish it to the container registry as configured in the `pom.xml` file, and then automatically https://docs.kalix.io/services/deploy-service.html#_deploy[deploy the service] to Kalix using `kalix`:
+. Use the `deploy` target to build the container image, publish it to the container registry as configured in the `pom.xml` file, and use the target `kalix:deploy` to automatically deploy the service to Kalix:
 +
+[source,command line]
 ----
-mvn deploy
+mvn deploy kalix:deploy
 ----
+IMPORTANT: If you time stamp your image. For example, `<dockerTag>${project.version}-${build.timestamp}</dockerTag>` you must always run both targets in one pass, i.e. `mvn deploy kalix:deploy`.
+You cannot run `mvn deploy` first and then `mvn kalix:deploy` because they will have different timestamps and thus different `dockerTag`s. Making it impossible to reference the image in the repository from the second target.
```

Comment:
Not entirely sure how it works, but maybe worth pulling this caveat out into a "partial" and use that instead of repeating it in so many places?

Reply:
Good idea

Reply:
done